### ISSUE TITLE
KAS ECC SSC Scope

### ISSUE THREAD

**Issue by swkeypair** [usnistgov/ACVP-Server#48]
The KAS ECC SSC tests include key validity tests in the scope. This implies that the validity check must be implemented within the boundary of the KAS ECC SSC primitive, rather than within the greater scope of the module. The assurances may take place at a different point in time: when a key is generated; when a public key is received, etc.

The [ACVP KAS ECC SSC specification Section 6.1](https://usnistgov.github.io/ACVP/draft-hammett-acvp-kas-ssc-ecc.html#name-requirements-covered), 4th bullet says this:
> SP 800-56Ar3 - 5.6 Key-Pair Generation. While Key-Pairs are used in each KAS scheme, the generation of said key-pairs is out of scope for KAS testing. Random tests from the VAL groups, MAY inject bad keypairs that the IUT MUST be able detect. These random tests are only present in groups given appropriate assurance functions see: Section 3.3 

This seems to exceed the scope of testing implied for this primitive.

**Comment by Kritner** [comment]
Yeah this language should be update, trusted third parties are able to provide key pairs so the IUT would not necessarily be able to check the validity of a key pair.
